Cannot Compile C++ Project in UnrealEngine 4.24.2 with the Plugin installed

GitSourceControlProvider.cpp(22): fatal error C1083: Cannot open include file: 'IPluginManager.h': No such file or directory LogMainFrame: MainFrame: Module compiling took 66.394 seconds

--- a/Source/GitSourceControl/Private/GitSourceControlProvider.cpp
+++ b/Source/GitSourceControl/Private/GitSourceControlProvider.cpp
@@ -19,7 +19,7 @@
 #include "ScopedSourceControlProgress.h"
 #include "SourceControlHelpers.h"
 #include "SourceControlOperations.h"
-#include "IPluginManager.h"
+#include "Interfaces/IPluginManager.h"
 
 #define LOCTEXT_NAMESPACE "GitSourceControl"
 

